bundles/org.openhab.binding.resol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="resol"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="vbuslan"/>
+		</supported-bridge-type-refs>
+
+		<label>Resol Device</label>
+		<description>Solar or system controller (or any other real device on the VBus) from Resol.</description>
+	</thing-type>
+
+	<thing-type id="emulatedEM">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="vbuslan"/>
+		</supported-bridge-type-refs>
+
+		<label>Emulated EM Device</label>
+		<description>Emulation of an Extension Module (EM) device which can be connected through the VBUS to Resol controllers
+			which support the EM devices. Replaces a physically available EM by openHAB.</description>
+		<channels>
+			<channel id="relay_1" typeId="relay"/>
```

Comment:
You can give the enumerated channels a label so they are better recognizable in the ui:
```suggestion
			<channel id="relay_1" typeId="relay">
			    <label>Relay 1</label>
			</channel>
```

Reply:
ok, should in this case the Label in the channel-type be removed?

Reply:
I have added the individual labels, but should the label in the channel-type stay as it is never used now?